### ISSUE TITLE
Request: option to disable video display

### ISSUE THREAD

**Issue by paulscottrobson** [commanderx16/x16-emulator#236]
Possibly via -scale = 0 ? ; reason ; scripted code testing. It works fine but of course pops up a window every so often.

**Comment by nigels-com** [comment]
Is the requirement to not _show_ the display?  Hiding the window would suffice?

**Comment by paulscottrobson** [comment]
Yep, that would be fine. Want it to run in the background without grabbing the focus or flashing the window up :)

**Comment by CJLove** [comment]
On Linux a workaround would be to use the X11 virtual frame buffer `xvfb` rather than the actual X Windows display.

Install the Xvfb x11 server (distro-specific).  For Fedora this works:
```
sudo dnf install xorg-x11-server-Xvfb.x86_64
```

Run the emulator and your desired program using `xvfb-run`:
```
xvfb-run x16emu -prg factorial_test.prg  -echo  -run
```
Note: this probably requires that the test program forces the emulator to exit (e.g. via `sys $ffff` or equivalent)
